ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionConfirmationView.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveUI
+import Shared
+import SwiftUI
+
+public struct BraveVPNRegionConfirmationView: View {
+  var regionCountry: String?
+  var regionCity: String?
+  var regionCountryFlag: Image?
```

Comment:
these can't be `let`s?

Reply:
Nice catch fix at https://github.com/brave/brave-core/pull/24198/commits/9e249c4b30c09ca2b440f34b2abc09264ebf3f68